scripts/run-passive.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# This script collects all markdown files in the documentation repositories, and
+# filters them through the blacklist, and then runs the `passive.sh` script on
+# them.
+#
+# Usage:
+#
+# ./scripts/run-passive.sh [--verbose]
+#
+# Note:
+#
+# The script automatically reads in the .passive-blacklist from the root
+# directory, and applies the entries as a filter to a grep expression. Have a
+# look at the `.passive-blacklist` file in the root directory for an example of
+# how it works.
+#
+
+
+# List all files not in caught by the blacklist
+files=$(find . -type f -regex './[0-9]+.*\.md' | grep -vFf .passive-blacklist)
+
+EXIT_CODE=0
+
+if [[ "$1" == "--verbose" ]]; then
+  VERBOSE="--verbose"
+fi
+
+for file in $files; do
+  if ! ./scripts/passive.sh $VERBOSE $file; then
+    EXIT_CODE=1
+  fi
+done
```

Comment:
Why don't just:
```suggestion
if ! ./scripts/passive.sh $VERBOSE $files; then
  EXIT_CODE=1
fi
```

AFAICS, `passive.sh` script accepts a list of files.

Reply:
Excellent suggestion. I was thinking of doing this initially. However, the `passive.sh` script accepts a list of files, but it exits on the first file which does not pass. I was thinking about this a bit. And I do think this is fair behaviour. So therefore, we call the files individually, so that in CI, all the failures show up. When running this from the CLI locally, having the first one fail, is sort of nice, as it allows you to weed out the failures one by one.

I'm open to other approaches though

Reply:
That makes sense and it is a good idea!

We actually had the same situation with the AUTOVERSION tool in this repo (i.e. CI exited on first failure) and was a bad experience (you as a PR author don't get the full picture at once).

So yes, leave as it is :+1: